pilz_control/test/acceptance_test_acceleration_limit.py

```diff
@@ -24,10 +24,11 @@
 from holding_mode_service_wrapper import HoldingModeServiceWrapper
 from trajectory_dispatcher import TrajectoryDispatcher
 
-# Change the following two lines if you run a different robot
-CONTROLLER_NS = '/prbt'
-CONTROLLER_NAME = 'manipulator_joint_trajectory_controller'
+DEFAULT_CONTROLLER_NS = '/prbt'
```

Comment:
I very much like the idea that we now are able to execute these tests without real hardware. I, therefore, suggest that we add a comment in the `acceptance_test_acceleration_limit.md` and `acceptance_test_speed_monitoring.md` on how to start the tests without real hardware. In other words, I suggest to add the things that you already stated in your PR description. What do you think? 

We could also include these tests in the CI? What you think about that?

Reply:
Thanks. I added them.

I think, before we can add these test to the CI, we have to pay special attention if they can run reliably on a build server. But it is a good point. This could be a new maintenance task.

Reply:
Sounds good to me.